### ISSUE TITLE
Add an ability to open related tables via right sidebar

### ISSUE THREAD

**Issue by bocharsky-bw** [TablePlus/TablePlus#1678]
1. Which driver are you using and version of it (Ex: PostgreSQL 10.0): MariaDB 10.2

2. Which TablePlus build number are you using (the number on the welcome screen, Ex: build 81): v3.1.0 (290)

3. The steps to reproduce this issue:

I want to suggest an improvement in the UI. IN table results I can click on an arrow near the foreign ID to open the entry in another related table, but I can't do it via the right sidebar. Would be convenient to have the same arrow that will link me to the related table and open the specific entry via the right sidebar, because thanks to the sidebar I can quickly filter columns and find the one I needed, then open the related table from it. Here's the screenshot:
<img width="362" alt="Screenshot 2020-01-24 at 13 21 18" src="https://user-images.githubusercontent.com/3317635/73065865-42d8f300-3ead-11ea-8762-78f16bc74270.png">

Basically, I just suggest duplicating the same link arrow from the main results set to the right sidebar

**Comment by oomathias** [comment]
Even better would be to see the row inline:

<img width="571" alt="image" src="https://user-images.githubusercontent.com/465790/77201682-d103db00-6ae4-11ea-9c1c-9483e9fadd1a.png">

Screenshot from Postico

**Comment by huyphams** [comment]
I just wanna let you know that we can open the linked table via command + enter. hope it helps
<img width="1830" alt="Screen Shot 2020-09-27 at 4 50 15 PM" src="https://user-images.githubusercontent.com/3654622/94361849-868d9780-00e1-11eb-9346-655b675ab924.png">

**Comment by bocharsky-bw** [comment]
Hey @huyphams ,

Thanks for replying! It's good to know! But unfortunately, it didn't help too much as it works only in the main window however I'm talking about the right sidebar. With the right sidebar, I can quickly filter columns (sometimes my tables may have a lot of columns to look for a specific column manually), and then would be nice to be able to open that row in another tab directly from the right sidebar.

As you can see, we have that "arrow" link in the main window:
<img width="118" alt="Screenshot 2020-09-28 at 17 48 01" src="https://user-images.githubusercontent.com/3317635/94447815-e43de480-01b2-11eb-8432-5ea8e4b08a5c.png">

But we don't have such link in the right sidebar:
<img width="261" alt="Screenshot 2020-09-28 at 17 48 38" src="https://user-images.githubusercontent.com/3317635/94447858-f3bd2d80-01b2-11eb-8ad5-cb4f8c3892f3.png">

I hope I explained well
